timeboost/auctioneer.go

```diff
@@ -341,6 +353,11 @@ func (a *AuctioneerServer) resolveAuction(ctx context.Context) error {
 	opts := copyTxOpts(a.txOpts)
 	opts.NoSend = true
 
+	// Once resolveAuction returns, we acknowledge all bids to remove them from redis.
+	// We remove them unconditionally, since resolveAuction retries until the round ends,
+	// and there is no way to use them after the round ends.
+	defer a.acknowledgeAllBids(ctx, upcomingRound)
```

Comment:
Is it fine if this function exits early because of an error and we still ack all bids?